frontend/src/components/dashboard/create/telegram/TelegramCredentials.tsx

```diff
@@ -371,10 +370,10 @@ const TelegramCredentials = ({
                 <ErrorBlock>{errorMessage}</ErrorBlock>
               </StepSection>
 
-              <ButtonGroup>
-                <NextButton disabled={!hasCredential} onClick={onNext} />
-                <TextButton onClick={onPrevious}>Previous</TextButton>
-              </ButtonGroup>
+              <NextButton
```

Comment:
Previous button is missing here. 

<img src='https://user-images.githubusercontent.com/3666479/98773918-dec7f100-2424-11eb-87fe-6c469cff2449.png' width=400 />